src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageNamespacesConfiguration.cs

```diff
@@ -9,14 +9,40 @@ namespace NuGet.Configuration
 {
     public class PackageNamespacesConfiguration
     {
+        /// <summary>
+        /// Max allowed length for package Id.
+        /// In case update this value please update in src\NuGet.Core\NuGet.Packaging\PackageCreation\Utility\PackageIdValidator.cs too.
+        /// </summary>
+        internal static int PackageIdMaxLength { get; } = 100;
+
         /// <summary>
         /// Source name to package namespace list.
         /// </summary>
-        public Dictionary<string, IReadOnlyList<string>> Namespaces { get; }
+        internal Dictionary<string, IReadOnlyList<string>> Namespaces { get; }
+
+        private Lazy<SearchTree> SearchTree { get; }
+
+        /// <summary>
+        /// Indicate if any packageSource exist in package namespace section
+        /// </summary>
+        public bool IsNamespacesEnabled { get; }
```

Comment:
`AreNamespacesEnabled`?

https://source.dot.net/#q=Are

Reply:
Thank you for suggestion. Applied.